cms/djangoapps/contentstore/courseware_index.py

```diff
@@ -175,8 +181,20 @@ def index_item(item, skip_index=False, groups_usage_info=None):
                 return
 
             item_content_groups = None
+
+            if item.category == "split_test":
```

Comment:
Why isn't this code in the "split_test" `index_dictionary` method? 

Seems like that's because we are trying to update `groups_usage_info` here, which might be the right thing, but I'm hoping you can explain why.